purchase_discount/tests/test_purchase_discount.py

```diff
@@ -7,8 +7,8 @@
 from odoo import fields
 
 
-class TestPurchaseOrder(common.HttpCase):
-    at_install = False
+class TestPurchaseOrder(common.TransactionCase):
+    at_install = True
```

Comment:
Please enable `post_install` instead for avoiding conflicts when testing with other modules, but indeed both type of tests are disabled right now.